doc/quickstart.md

```diff
@@ -2,8 +2,12 @@
 
 ## Prerequisites
 
-1. Kubernetes or Openshift cluster running on bare metal nodes.
-2. Multus-cni is deployed as default CNI plugin, and there is a default CNI plugin (flannel, openshift-sdn etc.) available for Multus-cni.
+1. A [supported](https://docs.openshift.com/container-platform/4.2/networking/multiple_networks/configuring-sr-iov.html#supported-devices_configuring-sr-iov) SRIOV hardware on the cluster nodes.
```

Comment:
This is linking an old version of the documentation. Could we list them here directly  based on https://github.com/openshift/sriov-network-operator/blob/0c0ad1c54fc06c591c52b2608f06b0ec7d945749/pkg/webhook/validate.go#L24?

Reply:
yeah sure. Done.